.github/workflows/nightly-pax-build.yaml

```diff
@@ -58,11 +68,26 @@ jobs:
           BUILD_DATE=$(TZ='US/Los_Angeles' date '+%Y-%m-%d')
           echo "BUILD_DATE=${BUILD_DATE}" >> $GITHUB_OUTPUT
 
+      - name: Set base image
+        id: base-image
+        shell: bash -x -e {0}
+        run: |
+          if [[ -z "${{ inputs.BASE_IMAGE }}" ]]; then
+            BASE_IMAGE_AMD64=${{ env.DOCKER_REGISTRY }}/jax:mealkit
+            BASE_IMAGE_ARM64=${{ env.DOCKER_REGISTRY }}/jax:mealkit
+          else
+            BASE_IMAGE_AMD64=${{ inputs.BASE_IMAGE }}-amd64-mealkit
```

Comment:
I am following the convention @DwarKapex introduced. I understand the reason he introduced this was because we don't provide a multiarch tag within `jax-toolbox-internal` if publish=false. We could provide such a tag in `jax-toolbox-internal` even if publish=false and that should simplify this. How does that sound?